Src/Localization/Russian/ButtonTutorial.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
```

Comment:
Adding these files doesn't seem to be related to `Change window title Help -> Open-Shell Help`.
Please, add unrelated changes in separate commit.